service/keyv2/key.go

```diff
@@ -119,6 +119,10 @@ func MasterInstanceType(customObject v1alpha1.AWSConfig) string {
 	return instanceType
 }
 
+func PeerAccessRoleName(customObject v1alpha1.AWSConfig) string {
+	return fmt.Sprintf("%s-peer-access", ClusterID(customObject))
```

Comment:
Maybe this could be `-vpc-peer-access`? A bit longer but makes it clear it's for the VPC peering.

Reply:
Sure, done